testing/trino-testing/src/main/java/io/trino/testing/datatype/SqlDataTypeTest.java

```diff
@@ -90,7 +90,7 @@ private void verifySelect(QueryRunner queryRunner, Session session, TestTable te
         QueryAssert assertion = assertThat(queryAssertions.query(session, "SELECT * FROM " + testTable.getName()));
         MaterializedResult expected = queryRunner.execute(session, testCases.stream()
                 .map(TestCase::getExpectedLiteral)
-                .collect(joining(",", "VALUES (", ")")));
+                .collect(joining(",", "VALUES ROW(", ")")));
```

Comment:
I tried to follow the discussion, but I'm not sure I understand the purpose of this.

`VALUES (1, 2)` and `VALUES ROW(1, 2)` are the same, so this looks like a no op.

Reply:
This change affects in case of ROW type test. 
```
trino> VALUES(ROW(1,2));
 _col0 | _col1
-------+-------
     1 |     2

trino> VALUES(ROW(ROW(1,2)));
 _col0
--------
 {1, 2}
```